slices/openjdk-11-jdk-headless.yaml

```diff
@@ -0,0 +1,162 @@
+package: openjdk-11-jdk-headless
+
+essential:
+  - openjdk-11-jdk-headless_copyright
+
+slices:
+
+  # OpenJDK binaries slice
+  core:
+    essential:
+      - libc6_libs
+      - openjdk-11-jre-headless_core
+      - openjdk-11-jre-headless_javac
+      - zlib1g_libs
+    contents:
+      /usr/lib/jvm/java-11-openjdk-*/bin/javac:
+      /usr/lib/jvm/java-11-openjdk-*/bin/javadoc:
+      /usr/lib/jvm/java-11-openjdk-*/bin/javap:
+      /usr/lib/jvm/java-11-openjdk-*/bin/jdeprscan:
+      /usr/lib/jvm/java-11-openjdk-*/bin/jdeps:
+      /usr/lib/jvm/java-11-openjdk-*/bin/jimage:
+      /usr/lib/jvm/java-11-openjdk-*/bin/serialver:
+
+  # excludes jaotc - this requires binutils
+  standard:
+    essential:
+      - openjdk-11-jdk-headless_core
+      - openjdk-11-jre-headless_standard
```

Comment:
Nitpick: just the `standard` is fine.

Reply:
This is core *jdk* + standard *jre*, so we need both.

Reply:
Oh, right! :eyeglasses: